rollup.config.js

```diff
@@ -36,6 +37,7 @@ export default {
       exclude: 'node_modules/**',
       presets: ['@babel/env', '@babel/preset-react']
     }),
+    glslify({ basedir: 'src/layers/XRLayer' }),
```

Comment:
Is this for the color maps? "gl-modules" or something?

Reply:
Yup.